pkg/minikube/machine/cache_images.go

```diff
@@ -131,20 +127,38 @@ func CacheAndLoadImages(images []string) error {
 		return err
 	}
 	defer api.Close()
-	cc, err := config.Load()
-	if err != nil {
-		return err
-	}
-	h, err := api.Load(cc.Name)
+	profiles, _, err := config.ListProfiles() // need to load image to all profiles
 	if err != nil {
-		return err
+		return errors.Wrap(err, "list profiles")
 	}
-
-	runner, err := CommandRunner(h)
-	if err != nil {
-		return err
+	for _, p := range profiles { // adding images to all the profiles
```

Comment:
This feels weird: what happens if a new profile is added? It seems like the cache should be universal.

Reply:
can you clarify what you mean ?